site/_includes/getting-started-sidebar/master/sidebar.html

```diff
@@ -0,0 +1 @@
+This is the getting started sidebar.
```

Comment:
This is the sidebar include partial for pages in the `getting-started` category.